moviepy/audio/io/ffmpeg_audiowriter.py

```diff
@@ -135,12 +135,32 @@ def __del__(self):
 
     # Support the Context Manager protocol, to ensure that resources are cleaned up.
 
+    def close(self):
+        if self.proc:
+            self.proc.stdin.close()
+            self.proc.stdin = None
+            if self.proc.stderr is not None:
+                self.proc.stderr.close()
+                self.proc.stdee = None
+            # If this causes deadlocks, consider terminating instead.
+            self.proc.wait()
+            self.proc = None
+
+    def __del__(self):
+        # If the garbage collector comes, make sure the subprocess is terminated.
+        self.close()
+
+    # Support the Context Manager protocol, to ensure that resources are cleaned up.
+
     def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
+    def __exit__(self, exc_type, exc_value, traceback):
```

Comment:
this es repeated, right?

Reply:
You are right. Well spotted. That is an error. (It is of no consequence, but one should be deleted.)